.github/workflows/check-for-new-cascadia.yml

```diff
@@ -6,25 +6,45 @@ on:
 
 jobs:
   build:
-    runs-on: ubuntu-18.04
+    runs-on: ubuntu-20.04
     steps:
-    - uses: actions/checkout@v2        
+    - uses: octokit/request-action@v2.x
```

Comment:
I like the way you've added names to the other steps, can you add to this too for consistency